api-reference/10 UI Components/GridBase/1 Configuration/stateStoring/customSave.md

```diff
@@ -81,7 +81,7 @@ If you need to save and load the state from a remote storage, use the following
                 body: data ? JSON.stringify(data) : null
             });
             return httpClient.request(req)
-                .toPromise();
+                .lastValueFrom();
```

Comment:
it's not a method, it's top-level function. See https://github.com/DevExpress/devextreme-demos/pull/1716